test/unit/data/events/sanitise-events.test.js

```diff
@@ -54,9 +54,9 @@ describe('sanitise events', () => {
     expect(result[0].status.default).toBe(true)
   })
 
-  test('should add last updated as last event time formatted as string', () => {
+  test('should add last updated as last event time formatted as string in London timezone', () => {
```

Comment:
Because that's the timezone we're in and therefore the one that will make sense to users.  If we don't change it to London, it will show as UTC time which will be an hour out for half the year

Reply:
Timezones are weird